dartagnan/src/test/resources/spirv/array.spv.dis

```diff
@@ -1,4 +1,4 @@
-; @Input: %v3v = Array {0, 0, 0}
+; @Input: %v3v = {0, 0, 0}
 ; @Output: forall (%v3v[0]==0 and %v3v[8]==1 and %v3v[16]==2)
```

Comment:
This is just a question out of curiosity: do you intend to have byte-granular accesses to vectors?
I would have thought using indices `0, 1, 2` is more natural than using the byte-offsets `0, 8, 16`.
Maybe this is just a temporary solution?

Reply:
It was from some old tests, updated to use indices now